waydroid-patches/base-patches-30/frameworks/base/0036-Force-hide-status-bar-in-multi-windows-mode.patch

```diff
@@ -52,7 +52,7 @@ index 81378831adbc..4e68edc08aea 100644
 +    public static boolean shouldForceHide(@InternalInsetsType int type) {
 +        return (type == ITYPE_CAPTION_BAR ||
```

Comment:
hold on, this was never meant to be `ITYPE_CAPTION_BAR`, it was meant to be `ITYPE_CLIMATE_BAR`.

Interesting that this didn't totally break the caption bar...